cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,33 @@
+if(NOT EXISTS "@CMAKE_BINARY_DIR@/install_manifest.txt")
+	message(FATAL_ERROR "Cannot find install manifest: @CMAKE_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
+foreach(file ${files})
+	message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
+	if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
+		exec_program(
+			"@CMAKE_COMMAND@" ARGS "-E rm -r \"$ENV{DESTDIR}${file}\""
+			OUTPUT_VARIABLE rm_out
+			RETURN_VALUE rm_retval
+			)
+		if(NOT "${rm_retval}" STREQUAL 0)
+			message(FATAL_ERROR "Problem when removing $ENV{DESTDIR}${file}")
+		endif()
+	else(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
+		message(STATUS "File $ENV{DESTDIR}${file} does not exist.")
+	endif()
+endforeach()
+
+set(OSC_LIB_DIR @CMAKE_INSTALL_PREFIX@/lib/osc)
+set(OSC_SHARE_DIR @CMAKE_INSTALL_PREFIX@/share/osc)
```

Comment:
Missing `$ENV{DESTDIR}`, I think.

Reply:
this was added to remove the empty directories which remain in /usr/local/lib and /usr/local/share/. $ENV{DESTDIR} is not needed for this.  Can you elaborate a bit, please ?

Reply:
Well, try it yourself:

```
DESTDIR=/tmp/test make install
DESTDIR=/tmp/test make uninstall
```

It won't remove `/tmp/test/usr/local/lib/osc` and `/tmp/test/usr/local/share/osc`.

Reply:
fixed this in a new commit